src/index.html

```diff
@@ -11,7 +11,191 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <div>
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            id="surname"
+            autocomplete="off"
+            name="surname"
+            required
+          >
+        </div>
+
+        <div>
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            id="name"
+            autocomplete="off"
+            name="name"
+            required
+          >
+        </div>
+
+        <div>
+          <label for="age">How old are You?</label>
+          <input
+            type="number"
+            id="age"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+            required
+          >
+        </div>
+
+        <div>
+          <label for="date_of_birth">Full date of birth:</label>
+          <input
+            type="date"
+            id="date_of_birth"
+            name="date_of_birth"
+            required
+          >
+        </div>
+
+        <div>
+          <label for="term">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            id="term"
+            required
+            name="term"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration</legend>
+
+        <div>
+          <label for="email">E-mail:</label>
+          <input
+          type="email"
+          id="email"
+          placeholder="email@example.com"
+          required
+          minlength="5"
+          maxlength="50"
+          name="email"
+          >
+        </div>
+        <div>
+          <label for="pass">Password:</label>
+          <input
+          type="password"
+          id="pass"
+          name="pass"
+          minlength="5"
+          maxlength="30"
+          required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+
+        <div>
+          Do you love cats?
+
+          <input
+          type="radio"
+          id="yes"
+          name="cat"
+          >
+          <label for="yes">Yes</label>
+
+          <input
+          type="radio"
+          id="no"
+          name="cat"
+          >
```

Comment:
```suggestion
          <input
            type="radio"
            id="no"
            name="cat"
          >
```
fix it all over your code